tests/integration/test_keeper_nodes_remove/test.py

```diff
@@ -125,6 +125,10 @@ def test_nodes_remove(started_cluster):
 
         node2.stop_clickhouse()
     finally:
+        for i in range(100):
+            zk_conn.delete("/test_two_" + str(i))
+            zk_conn2.delete("/test_two_" + str(100 + i))
```

Comment:
Flakiness here will be fixed in https://github.com/ClickHouse/ClickHouse/pull/77749